FutureNorthants/northants-design-system
# Add births, deaths, marraiges icons

Added new icons for:

- Birth certificate
- Ceremony
- Death certificate
- Give notice
- Register birth
- Register death
- Venue

## Testing
- Checkout this branch and run `npm run dev`
- Go to Iconography -> Service icons, and then check the new icons appear as expected and hover over to check the hover state
